client/starwhale/api/_impl/wrapper.py

```diff
@@ -23,6 +23,8 @@ def close(self) -> None:
 
     def _log(self, table_name: str, record: Dict[str, Any]) -> None:
         with self._lock:
+            if table_name not in self._writers.keys():
```

Comment:
`if table_name not in self._writers.keys():` -> `if table_name not in self._writers:` ?